.azure-pipelines/azure-pipelines-UpgrateVersion.yml

```diff
@@ -16,6 +16,18 @@ schedules:
 
 pool: sonicbld
 
+parameters:
+  - name: jobFilters
```

Comment:
https://dev.azure.com/mssonic/build/_build/results?buildId=12798&view=results
There is some format error.

Reply:
The issue should have been fixed, it is colon issue.